File: Pascal/Niveau 1/2 – Répétitions d'instructions/Mont Kailash.pas
program Solution;
uses Robot;
var
   i: LongInt;
   j: LongInt;
begin
   for i := 1 to 108 do
   begin
      for j := 1 to 13 do
      begin
         haut;
      end;
      for j:= 1 to 13 do
      begin
         droite;
      end;
      for j:= 1 to 13 do
      begin
         bas;
      end;
      for j:= 1 to 13 do
      begin
         gauche;
      end;
   end;
end.

File: OCaml/Niveau 1/2 – Répétitions d'instructions/Mont Kailash.ml
open Robot;;

for i = 1 to 108 do
  for j = 1 to 13 do
    haut();
  done;
  for j = 1 to 13 do
    droite();
  done;
  for j = 1 to 13 do
    bas();
  done;
  for j = 1 to 13 do
    gauche();
  done;
done;